build/_k-test.shade

```diff
@@ -58,7 +59,10 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                Dnx("test" + testArgs, projectFolder);
+                if (!IsLinux || string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO))
```

Comment:
BTW the reasoning for this is: options are generally **off** by default, so we want only very particular settings to turn an option **on**. All other values do nothing.